### PR TITLE
Sync SEO Angular components with experiments changes 

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Sync experiment changes in `dot-edit-page-state-controller-seo.component` and `dot-edit-page-view-as-controller-seo.component.html`

* Experiments affected tasks: #25645 and #25688

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.spec.ts [hunk 1]
```diff
@@ -158,7 +171,8 @@ describe('DotEditPageStateControllerSeoComponent', () => {
                 expect(dotTabButtons).toBeDefined();
                 expect(dotTabButtons.options).toEqual([
                     { label: 'Edit', value: 'EDIT_MODE', disabled: false },
-                    { label: 'Preview', value: 'PREVIEW_MODE', disabled: false }
+                    { label: 'Preview', value: 'PREVIEW_MODE', disabled: false },
+                    { label: 'Live', value: 'ADMIN_MODE', disabled: false }
```
Comment (manuelrojas) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.spec.ts:175]: @hmoreras We don't have live anymore so I wonder if we can get ride of it?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts [hunk 2]
```diff
@@ -203,7 +206,7 @@ we want to show the lock off so the new user can steal the lock
     }
 
     private getStateModeOptions(pageState: DotPageRenderState): SelectItem[] {
-        const items = this.variant ? this.getModesBasedOnVariant() : ['edit', 'preview'];
+        const items = this.variant ? this.getModesBasedOnVariant() : ['edit', 'preview', 'live'];
```
Comment (manuelrojas) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts:209]: @hmoreras Same as before, we removed Live in the new Toolbar

**Review by zJaaal — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.spec.ts (hunk 1 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.spec.ts]: done

File: core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts (hunk 2 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts]: done

**Review by manuelrojas — APPROVED** [review]

**Pull request merged**